tests/unit/accounts/test_services.py

```diff
@@ -379,6 +379,22 @@ def test_add_email_rate_limited(self, user_service, metrics, remote_addr):
             )
         ]
 
+    def test_add_email_bypass_ratelimit(self, user_service, metrics, remote_addr):
```

Comment:
question: should this test assert that email is added successfully as well, not only that the limiter wasn't called?

Reply:
67e26a350ebbbe270899b6ee19a1aaa7b0ec7227